homework-submission/hw-todolist/src/components/TodoList/index.jsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+function TodoList({ todos, deleteTodo, markTodo }) {
+  return (
+    <table>
+      <tr>
+        <th>Todo</th>
+        <th>Deadline</th>
+        <th>Remove</th>
+        <th>Complete</th>
+      </tr>
+
+      {todos.map(todo => {
+        return (
+          <tr
+            style={
+              todo.done === true ? { textDecoration: 'line-through' } : { textDecoration: 'none' }
+            }
+          >
+            <td>{todo.description}</td>
+            <td>{todo.deadline}</td>
+            {'      '}
+            <td>
+              <button onClick={() => deleteTodo(todo.id)}>Delete todo</button>
+            </td>
+            <td>
+              <button onClick={() => markTodo(todo.id, todo.done)}>Mark todo</button>
```

Comment:
Would be cool if the text of the 'Mark todo' button is 'Mark done' at fist and changes to 'Mark todo' once a todo is done